src/core/containers/HandleLogin/index.js

```diff
@@ -33,7 +34,8 @@ class HandleLogin extends React.Component {
   }
 
   render() {
-    const { code, state } = this.props.location.query;
+    const { location } = this.props;
+    const { code, state } = location.query;
     const { error } = this.state;
```

Comment:
I think the only way to use redux is to use redux for all state management. It just makes the tooling so much more useful (time travel, debugging, etc). I filed a bug for this change: https://github.com/mozilla/addons-frontend/issues/1064